acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -64,6 +64,15 @@
   sitepp_content = generate_manifest(test_resources)
   assertion_code = generate_assertions(test_resources)
 
+  refutation_resources = test_resources.collect do |assertion_group|
```

Comment:
If the test is only looking at the `syslog` file content itself - and not the rest of the beaker log - the `assert_match` calls mostly (maybe all?) fail.  For example, the line with 'Sensitive [value redacted]' line will be written to the console debug output for the 'puppet agent' command but is not written to the 'syslog'.  I believe this is because 'puppet agent --debug' by default outputs to the console unless '--logdest syslog' is added to the command line.  If '--logdest syslog' were added to the command line, we'd get the extra lines like 'Sensitive [value redacted]' in the syslog but would not get them on the console, meaning that the console-related assertions in this test would fail.

I don't know of a way to have the Puppet agent "tee" the output both to the syslog and console.  Maybe that would be best for the purpose of this test if possible to do.  WDYT?